lib/brakeman/util.rb

```diff
@@ -124,6 +124,18 @@ def hash_access hash, key
     nil
   end
 
+  def call_has_param call, key
```

Comment:
I believe this method is equivalent to

```ruby
call? arg and call? arg.target and node_type? arg.target.target, :params
```

I don't think it should be a general purpose method living in `Util`.